tests/Classifai/Providers/Azure/ComputerVisionTest.php

```diff
@@ -98,4 +102,53 @@ public function test_set_image_meta_data() {
 		$meta       = wp_get_attachment_metadata( $attachment );
 		$this->assertNotFalse( $meta );
 	}
+
+	public function test_alt_text_option_reformatting() {
+		add_option( 'classifai_computer_vision', array() );
+
+		$options = array(
+			'valid'                 => false,
+			'url'                   => '',
+			'api_key'               => '',
+			'enable_image_captions' => '1',
+			'enable_image_tagging'  => '1',
+			'enable_smart_cropping' => 'no',
+			'enable_ocr'            => 'no',
+			'enable_read_pdf'       => 'no',
+			'caption_threshold'     => 75,
+			'tag_threshold'         => 70,
+			'image_tag_taxonomy'    => 'classifai-image-tags',
+		);
+
+		/** Test with `enable_image_captions` set to `1` */
```

Comment:
```suggestion
		// Test with `enable_image_captions` set to `1`
```

A couple of things here:

* `/**` indicated a docblock rather than a comment, however....
* the WPCS for single line comments is to use `//` rather than `/*`